### ISSUE TITLE
Scout and Windows 10 not working by default

### ISSUE THREAD

**Issue by eheddema** [scout-app/scout-app#211]
Hello,

We are a long time user of Scout and never had any issues! But Windows 10 has been released and now we had an issue with the Java runtime and the language setting. One of our clients found a workaround for this but we are of course hoping for a proper fix ;-).

Our workaround getting Scout to work in Windows 10.

- You need a 32bit Java 7 runtime. You may use http://ninite.com/air-java7 (from a Scout issue on GitHub) to get the right version or download it yourself.
- Windows needs to run in English using the settings panel to make US English default. Note that this changes your desktop language!
- A rather well hidden option should be used to set the language for non-Unicode programs to US English. Use the search task bar icon to find the control panel and open it. Then select region, administrative tab. The language for non-Unicode programs must be set to US English.

I later tried setting the regional settings back to Dutch and it still worked. I know Scout hasn't been updated actively and we will probably start to look at alternatives but we work with a big SASS framework and projects. So far Scout did the job nicely and worked a lot better than other Apps.

**Comment by TheJaredWilcurt** [comment]
Thanks for your support and usage of Scout. The current version will not be adapted for Windows 10, so thank you for consolodating the workaround into one place for others. A new version of Scout is in planning and will be using newer cross-platform technology that won't require Java or Air.

Sorry for the inconvenience.

**Comment by waschier-design** [comment]
When will the new version for win 10 be released?

**Comment by TheJaredWilcurt** [comment]
@waschier-design 

The main V2.0 design thread is here:

* https://github.com/mhs/scout-app/issues/186

I've been building a framework called [UGUI](https://github.com/UniversalGUI/UGUI) for a technology called NW.js. A while back I took a break from it to start in on making a basic, but functional prototype for Scout-App using that same NW.js technology. I was hoping to try out and learn some new NW.js tricks as I wasn't expecting there to be much overlap in the way Scout 2 would run behind the scenes and the way my framework functions.

Unfortunately, after exhausting all other routes, I found that the only way I could get it to work was via a method my framework is specifically designed to handle. And since the UGUI was nearing completion, I postponed Scout to finish it.

Then I found a major bug that required a major rewrite of UGUI. A few months later and I'm back to almost being done with UGUI, and therefore almost ready to start in earnest on Scout-App 2.0.

Since both UGUI and Scout-App are projects I work on for free in my spare time, it can be difficult to set and hit deadlines. So I can't give you a good answer on when Scout 2 will be released, especially since it isn't a solo project. It will however be Windows Vista, 7, 8, 8.1, and 10 compatible in the first prototype and ideally Ubuntu 12, Ubuntu 14, and OSX 10.9+ compatible by the time it's in beta.

If you are having trouble running Scout 0.7.1 on Windows 10, others have had success here:

* https://github.com/mhs/scout-app/issues/213
* https://github.com/mhs/scout-app/issues/187
